sdk/keyvault/azure-security-keyvault-keys/src/key_client.cpp

```diff
@@ -381,6 +381,59 @@ Azure::Response<KeyVaultKey> KeyClient::ImportKey(
   return Azure::Response<KeyVaultKey>(std::move(value), std::move(rawResponse));
 }
 
+Azure::Response<KeyRotationPolicy> KeyClient::GetKeyRotationPolicy(
+    std::string const& name,
+    Azure::Core::Context const& context) const
+{
+  // Request with no payload
+  auto request = CreateRequest(HttpMethod::Get, {_detail::KeysPath, name, "rotationpolicy"});
```

Comment:
move `rotationpolicy` to a constant. It is used in more than one place within this file